pkg/client/fixtures/with_ignore/.testgroundignore

```diff
@@ -0,0 +1,3 @@
+a
+b/c
```

Comment:
```suggestion
c
```

I think it'd be interesting to test if the above works. If we just have `b/c` here, then a naive implementation that doesn't turn lines here into regexes would still work but if we have only `c` then it wouldn't anymore. Does it make sense?

Reply:
Agreed, I updated the test!